.github/workflows/benchmark_asterinas.yml

```diff
@@ -69,17 +69,29 @@ jobs:
           make install_osdk
           bash test/benchmark/bench_linux_and_aster.sh ${{ matrix.benchmark }}
 
-    - name: Prepare threshold values
+    - name: Set up benchmark configuration
       run: |
         echo "Configuring thresholds..."
         ALERT_THRESHOLD=$(jq -r '.alert_threshold' test/benchmark/${{ matrix.benchmark }}/config.json)
         echo "ALERT_THRESHOLD=$ALERT_THRESHOLD" >> $GITHUB_ENV
+        echo "Configuring benchmark description..."
```

Comment:
Do not print "..." for very quick operations like "jq". Usually, "Doing something..." is printed if it is followed by a human observable latency. For example, `cargo test` would print "test <test_name> ..." because the test may run for an indefinite amount of time. This is not the case here. Instead print the values directly.